templates/Makefile.template

```diff
@@ -405,8 +405,8 @@
   SHARED_EXT_CSHARP = dll
   SHARED_PREFIX =
   SHARED_VERSION_CORE = -${settings.core_version.major}
```

Comment:
Why are we keeping the content of SHARED_VERSION_CORE but removing SHARED_VERSION_CPP and SHARED_VERSION_CSHARP?